packages/utils/scss/flex-grid/_flex.scss

```diff
@@ -1,19 +1,23 @@
-/// This is equivalent to `flex: 1 1 0%`. It sizes the item not based on its `width`/`height` properties, but based on the available space. This is similar to `flex: 1 1 auto` except it is allowed to shrink beyond its initial size.
+/// This is equivalent to `flex: 1 1 0%;`.
+/// @example flex: 1 1 0%;
 /// @name .k-flex-1
 /// @group flex
 /// @contextType css
 
-/// This is equivalent to `flex: 1 1 auto`. Beware, this is not the default value. It sizes the item based on its `width`/`height` properties, but makes it fully flexible so that they absorb any extra space along the main axis. If all items are either `flex: auto`, `flex: initial`, or `flex: none`, any remaining space after the items have been sized will be distributed evenly to the items with `flex: auto`.
+/// This is equivalent to `flex: 1 1 auto;`.
+/// @example flex: 1 1 auto;
 /// @name .k-flex-auto
 /// @group flex
 /// @contextType css
 
 /// This is the same as `flex: initial;` and the shorthand for the default value: `flex: 0 1 auto`. It sizes the item based on its `width`/`height` properties (or its content if not set). It makes the flex item inflexible when there is some free space left, but allows it to shrink to its minimum when there is not enough space. The alignment abilities or `auto` margins can be used to align flex items along the main axis.
```

Comment:
Shall we delete the above paragraph and replace it with This is equivalent to...